gitlint/utils.py

```diff
@@ -16,6 +16,17 @@
 import io
 import os
 import re
+import sys
+
+try:
+    if sys.version_info[1] < 5:
+        # Python3.4 doesn't have `Path.mkdir`'s `exists` option. Fall back.
+        # https://travis-ci.org/sk-/git-lint/jobs/382833916
+        from pathlib2 import Path  # python 2 backport
+    else:
+        from pathlib import Path
+except ImportError:
+      from pathlib2 import Path  # python 2 backport
```

Comment:
I would prefer to just use the backport all the time, as in `import pathlib2 as pathlib`. Then when I drop support for 2.7, 3.3 and 3.4 I can just remove the dependency and update the import.

Reply:
Switched to only using Pathlib2. I like this line of reasoning a lot.